lib/spack/spack/fetch_strategy.py

```diff
@@ -474,7 +500,14 @@ def check(self):
     @_needs_stage
     def expand(self):
         tty.debug(
-            "Source fetched with %s is already expanded." % self.url_attr)
+            "Moving source fetched with %s to %s." % (self.url_attr,
+                                                      self.stage.source_path))
```

Comment:
Do the users typically enable debugging messages?

Reply:
If they have issues, we ask the to run `spack —debug` to enable these.  But generally they just want to know that things are installed (and maybe why — i.e. what dependent caused the install, which we currently don’t show).

Reply:
They don't - it has to be enabled with `spack -d <command>`. We typically ask for the debug output of this command when they report an issue. I think it can relate to internals; if it shouldn't, then I think we should create a logging level where we *can* report internal state, for our own sanity when debugging an issue.

Reply:
Let's make it debug for now and revisit.  We had a conversation with @adamjstewart on another issue about simplifying `tty` to have *fewer* logging levels, but I actually think what we may need is to actually *use* a level between debug and `msg` more frequently. I think a lot of the things we currently print out should be saved in a log somewhere that can be automatically attached to issues.

See here: https://github.com/spack/spack/pull/11516#discussion_r285397597